src/plot_api/plot_api.js

```diff
@@ -312,7 +312,7 @@ Plotly.plot = function(gd, data, layout, config) {
 
         // show annotations and shapes
         Registry.getComponentMethod('shapes', 'draw')(gd);
-        Registry.getComponentMethod('annoations', 'draw')(gd);
```

Comment:
I wonder if we can run (most of) our tests in a mode where *all* components are registered, and `Registry.getComponentMethod` throws an error if it doesn't find what it's looking for, rather than returning `noop`?